### DISCUSSION TITLE
Bubble charts, update x-axis values and bubble size values

### DISCUSSION THREAD

**Discussion by zzzaaammm** [ShapeCrawler/ShapeCrawler#712] [Q&A]
Hi and thanks for a nice library. I wonder if it is possible to update x axis values ​​and bubble size values ​​for bubble charts? I note that it is possible to update the y values, but have not found a way to update the other dimensions

**Comment by AdamShakhabov** [comment]
Hello @zzzaaammm,

thank you for your feedback!

No, these APIs are not implemented yet.
